### PR TITLE
Update Editors, Authors, and Acknowledgements.

### PR DESCRIPTION
**Pull request by msporny**
This is the final **PROPOSED** list of Editors, Authors, and Acknowledgements. It was generated by pulling data from a variety of places (trying to be as data driven as possible). 

The Acknowledgements include every individual, in alphabetical order, that ever made a comment on an issue or PR over the years in Github.

The Editors are pulled directly from the contributor graph. The people in the top contributors by commits and lines changed are the Editors. There is a significant drop off in contribution after the 3rd person in the list for the DID Core specification.

https://github.com/w3c/did-core/graphs/contributors

The Authors list is a little more squishy. It's an amalgamation of contributions over the years that provided multiple significant architectural foundations as well as more recent and significant issue/PR feedback. 

I didn't include data from mailing list traffic, as the group didn't do most of its work via the mailing list. The data for issue/PR contributions are below. The `score` is the number of comments an individual made plus another score point for each 1KB of text that they wrote -- this is done in an attempt to balance out the contributions of people that write A LOT vs. those that fire off one liners):

                user |  score |   comments |    bytes |
              -----------------------------------------
             msporny |   1934 |       1363 |   585467 |
                OR13 |   1300 |        760 |   553464 |
         peacekeeper |    834 |        552 |   289422 |
             iherman |    637 |        401 |   241832 |
            dlongley |    509 |        248 |   267589 |
            jandrieu |    493 |        188 |   312979 |
          csuwildcat |    418 |        247 |   175584 |
          kdenhartog |    410 |        229 |   186056 |
             dhh1128 |    319 |        134 |   189907 |
            talltree |    319 |        174 |   149343 |
            agropper |    294 |        118 |   181040 |
              rhiaro |    280 |        186 |    97084 |
        SmithSamuelM |    245 |        110 |   138927 |
             TallTed |    209 |        124 |    87079 |
             ewelton |    204 |         60 |   147872 |
          selfissued |    200 |        149 |    52254 |
         brentzundel |    196 |        163 |    33888 |
             shigeya |    115 |         86 |    30619 |
             jricher |    107 |         74 |    34389 |
         jonnycrunch |    101 |         75 |    27493 |
        ChristopherA |     99 |         53 |    47702 |
               awoie |     78 |         49 |    29990 |
     dmitrizagidulin |     72 |         54 |    19185 |
            burnburn |     69 |         49 |    21166 |
            tplooker |     67 |         43 |    25161 |
      Oskar-Deventer |     64 |         42 |    22986 |
          philarcher |     57 |         25 |    33224 |
                 wyc |     50 |         32 |    19145 |
                 oed |     42 |         30 |    12512 |
              pknowl |     38 |         26 |    13107 |
            cboscolo |     31 |         21 |    10420 |
             Baha-sk |     27 |         13 |    14746 |
         mikelodder7 |     23 |         16 |     7767 |
             dhuseby |     20 |          8 |    12736 |
           CholoTook |     18 |         11 |     7698 |
      melvincarvalho |     18 |         10 |     8367 |
           davidlehn |     17 |         14 |     3762 |
           llorllale |     17 |         12 |     5904 |
          veikkoeeva |     17 |          6 |    11295 |
     darrellodonnell |     15 |          4 |    11765 |
           nalamillo |     14 |         11 |     3640 |
                gjgd |     13 |          8 |     5267 |
           troyronda |     12 |          9 |     4095 |
              mikeal |     11 |          7 |     4693 |
          grantnoble |     10 |          9 |     1462 |
           ewagner70 |     10 |          7 |     3806 |
        yancyribbens |      9 |          8 |     1700 |
         bumblefudge |      9 |          6 |     3659 |
          lrosenthol |      9 |          6 |     3365 |
              w3cbot |      9 |          9 |      774 |
        Crenshinibon |      8 |          7 |     1120 |
             clehner |      8 |          3 |     5687 |
             TimDaub |      8 |          3 |     5193 |
              mitfik |      7 |          3 |     5080 |
             DurandA |      7 |          5 |     2291 |
           TomCJones |      7 |          5 |     2537 |
              tahpot |      7 |          4 |     3352 |
              jbenet |      6 |          3 |     3441 |
        carsonfarmer |      6 |          5 |     1152 |
              deniak |      5 |          4 |     1787 |
            mavarley |      5 |          3 |     2122 |
             nikosft |      5 |          4 |     1838 |
              RieksJ |      4 |          2 |     2133 |
             glcohen |      4 |          3 |     1928 |
        mwherman2000 |      4 |          3 |     1243 |
                cabo |      4 |          3 |     1541 |
                mnot |      4 |          3 |     1667 |
        kimdhamilton |      3 |          3 |      348 |
           aljones15 |      3 |          2 |     1576 |
          BigBlueHat |      3 |          3 |      871 |
             jasnell |      3 |          2 |     1028 |
           marceljay |      3 |          2 |     1400 |
                 xfq |      3 |          3 |      478 |
                Fak3 |      3 |          3 |      117 |
             cihanss |      3 |          2 |     1901 |
           YueJing95 |      2 |          2 |      614 |
         PeterTheOne |      2 |          2 |      288 |
             mwklein |      2 |          2 |      763 |
         TelegramSam |      2 |          2 |      574 |
            gkellogg |      2 |          2 |      756 |
         timcappalli |      2 |          1 |     1142 |
            tralcanx |      2 |          2 |      600 |
               rvagg |      2 |          1 |     1683 |
             reschke |      2 |          2 |      117 |
          gimly-jack |      2 |          2 |      251 |
             ssstolk |      2 |          2 |      574 |
         samu-gataca |      2 |          1 |     1526 |
        thehenrytsai |      1 |          1 |      155 |
    kdenhartog-sybil1|      1 |          1 |       47 |
           azaroth42 |      1 |          1 |      950 |
               pelle |      1 |          1 |      255 |
            pensivej |      1 |          1 |      295 |
      David-Chadwick |      1 |          1 |      724 |
           ken-ebert |      1 |          1 |      476 |
           mirceanis |      1 |          1 |      584 |
           rknobloch |      1 |          1 |      182 |
       panickervinod |      1 |          1 |      215 |
                oare |      1 |          1 |      451 |
           haidedang |      1 |          1 |      127 |
       Identitywoman |      1 |          1 |      195 |
       mehranshakeri |      1 |          1 |       66 |
            sammotic |      1 |          1 |       98 |
         wolfmcnally |      1 |          1 |      163 |
          mooreT1881 |      1 |          1 |      824 |
            mprorock |      1 |          1 |      289 |
        pukkamustard |      1 |          1 |      101 |
             rxgrant |      1 |          1 |      924 |
            wseltzer |      1 |          1 |      343 |
     manicprogrammer |      1 |          1 |       31 |
            plehegar |      1 |          1 |      164 |
               zhanb |      1 |          1 |      151 |
          Diiaablo95 |      1 |          1 |      166 |
       praveensankar |      1 |          1 |      122 |
             ktobich |      1 |          1 |      368 |
             sondreb |      1 |          1 |      868 |
          Kalanamith |      1 |          1 |      282 |
          alenhorvat |      1 |          1 |      323 |

... and the spec contributor data at the time of the authoring of this PR:

image

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/780.html" title="Last updated on Jul 17, 2021, 7:50 PM UTC (ae5c53c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/780/65078db...ae5c53c.html" title="Last updated on Jul 17, 2021, 7:50 PM UTC (ae5c53c)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
...
+Nottingham, Darrell O'Donnell, Vinod Panicker, Dirk Porsche, Praveen, Mike
+Prorock, Drummond Reed, Julian Reschke, Yancy Ribbens, Justin Richer, Rieks,
+Mikeal Rogers, Evstifeev Roman, Troy Ronda, Leonard Rosenthol, Michael Ruminer,
+Markus Sabadello, Cihan Saglam, Samu, Rob Sanderson, Wendy Seltzer, Mehran
+Shakeri, Jaehoon (Ace) Shim, Samuel Smith, James M Snell, SondreB, Manu Sporny,
+Orie Steele, Shigeya Suzuki, Sammotic Switchyarn, Oliver Terbu, Joel
+Thorstensson, Tralcan, Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity
+Woman" Young, Eric Welton, Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel,
+cabo, ewagner70, gimly-jack, gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
+pukkamustard, rknobloch, ssstolk, tahpot, and zhanb.
```
Comment (TallTed) [inline index.html:6283]: I started by correcting my alphabetical placement (sorting on Thibodeau, not Jr); then I noticed there were several people following Zundel who should have been sorted on their single-name-as-family-name; then realized that several of these were github handles which might be better kept as `@`names like `@Yue`, but maybe that's not a github handle, so....

It seems the only safe alpha-sort for these names would be by treating each name as a string -- no special treatment for given vs family vs github vs other-nickname vs....

Or maybe there's [no safe name sort at all](https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/).

**Review by brentzundel — CHANGES_REQUESTED** [review]
First read through went too quickly, so I missed that Drummond was removed as an Editor.
I support folks who significantly contribute being added to the list of Editors and Authors, but need to be convinced that it is appropriate to remove someone who was named an Editor by the chairs.

**Comment by msporny** [comment]
> First read through went too quickly, so I missed that Drummond was removed as an Editor.

Yes, I expect the Chairs to discuss this (I was going to bring it up on the call tomorrow wrt. PRs of significance)... we might want to have the WG weigh in on this as well. I can outline the data-driven approach to get to this list to the group. I was going to hold off merging this until I got confirmation from @talltree (removed from Editors, but still at top of Authors list), @OR13 (added to Authors list due to issue/PR discussion contributions listed above), and @rhiaro (added to Editors list due to editorial work performed) and at least the Chairs on what they thought about the PR.

> I support folks who significantly contribute being added to the list of Editors and Authors, but need to be convinced that it is appropriate to remove someone who was named an Editor by the chairs.

I've tried to apply a fair and data driven approach to come up with the lists; I'm happy to elaborate more on the selections and ordering.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html:6283]: ... and if we add `@` to the beginning of all the github handles, they should sort first... which would be weird, 'cause who is `oare` and `mooreT1881` anyway and why are they at the front of the list?

I sorted by last word in name and then github handles in alphanum order (as you highlight, there is no really correct order). I didn't decloak people that didn't list their full name in their Github handle, unless they came to a public WG meeting... but maybe I shouldn't have done that... and so on, and so on for 30 minutes, @TallTed ... the result is the list above, and who knows if it's good enough. :)

... but, I did miss placing your name in the right place, so please feel free to do that (or I'll try to remember in my next pass). I also don't have a very strong opinion on sort order for single word names vs. multi word names... it's wrong any way you do it. :)

File: index.html [hunk 2]
```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
...
+Looker, Wolf McNally, Robert Mitwicki, Mircea Nistor, Grant Noble, Mark
+Nottingham, Darrell O'Donnell, Vinod Panicker, Dirk Porsche, Praveen, Mike
+Prorock, Drummond Reed, Julian Reschke, Yancy Ribbens, Justin Richer, Rieks,
+Mikeal Rogers, Evstifeev Roman, Troy Ronda, Leonard Rosenthol, Michael Ruminer,
+Markus Sabadello, Cihan Saglam, Samu, Rob Sanderson, Wendy Seltzer, Mehran
+Shakeri, Jaehoon (Ace) Shim, Samuel Smith, James M Snell, SondreB, Manu Sporny,
+Orie Steele, Shigeya Suzuki, Sammotic Switchyarn, Oliver Terbu, Joel
+Thorstensson, Tralcan, Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity
+Woman" Young, Eric Welton, Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel,
+cabo, ewagner70, gimly-jack, gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
```
Comment (kdenhartog) [inline index.html:6282]:
```suggestion
cabo, ewagner70, gimly-jack, gjgd, ktobich, mooreT1881, oare,
```

I don't remember what the purpose of this troll was about but I'm listed above by name so this account doesn't need to be double listed.

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:6283]: In my experience, single-word names are usually best treated as "floaters" that go in the column chosen for the overall sort (that is -- treat them as "first" if sorting by "first"; treat them as "last" if sorting by "last").

As to the github handles, I'd keep the `@`, and include a note saying that's what those are.  For sorting, I'd treat the handles as single-word names, and sort them by the handle minus the `@`.

(In my perfect world, I'd also include known github handles, maybe "Ted (@TallTed) Thibodeau Jr" or similar, with the overall sorting still by last name.)

fwiw, as you may also have discovered, I've found a spreadsheet tends to be the best tool for the bulk of this sort of processing (of which I've done a fair amount, over 30 years of volunteer and professional publishing efforts).  You can easily have an extra "sort" column alongside those for first, last, github, other, etc.; use formula(s) to build the "presentation" name; and then export that last column to a plain text doc for the final fluff of changing line breaks to comma+spaces.

Or just keep what you've got, modulo relocating my name.  ;-)

File: index.html (hunk 2 above)
Comment (TallTed) [inline index.html:6282]:
```suggestion
Jones, jonnycrunch, Gregg Kellogg, Michael Klein, Paul Knowles, David I. Lehn,
Charles Lehner, Michael Lodder, Dave Longley, Tobias Looker, Wolf McNally,
Robert Mitwicki, Mircea Nistor, Grant Noble, Mark Nottingham, Darrell O'Donnell,
Vinod Panicker, Dirk Porsche, Praveen, Mike Prorock, Drummond Reed, Julian
Reschke, Yancy Ribbens, Justin Richer, Rieks, Mikeal Rogers, Evstifeev Roman,
Troy Ronda, Leonard Rosenthol, Michael Ruminer, Markus Sabadello, Cihan Saglam,
Samu, Rob Sanderson, Wendy Seltzer, Mehran Shakeri, Jaehoon (Ace) Shim, Samuel
Smith, James M Snell, SondreB, Manu Sporny, Orie Steele, Shigeya Suzuki,
Sammotic Switchyarn, Oliver Terbu, Ted Thibodeau Jr, Joel Thorstensson, Tralcan,
Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity Woman" Young, Eric Welton,
Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel, cabo, ewagner70, gimly-jack,
gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
```

File: index.html [hunk 3]
```diff
@@ -68,20 +68,20 @@
...
       {
         name: "Manu Sporny",
         url: "http://manu.sporny.org/",
         company: "Digital Bazaar",
         companyURL: "https://digitalbazaar.com/",
         w3cid: 41758
       },
+      {
+        name: "Amy Guy",
+        url: "http://rhiaro.co.uk/",
```
Comment (rhiaro) [inline index.html:81]:
```suggestion
        url: "https://rhiaro.co.uk/",
```

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html:6282]: I saw the entry when putting the list together, so this is in there on purpose. :)

I'd like to leave `kdenhartog-sybil1` in the acknowledgements as an inside joke... the acknowledgements thank individuals, and this is a Decentralized Identifier specification... so having a sybil be in the list of individuals that we are thanking feels like the right level of irreverence... all is lost if we take ourselves too seriously in our path to SSI... and, well... if we can't have fun on the way, what's the point. :)

Comment (msporny) [reply index.html:6282]: > removed the extra `kdenhartog-sybil1`

See reasoning for not removing the extra `kdenhartog-sybil1`: https://github.com/w3c/did-core/pull/780#discussion_r668709801

Comment (TallTed) [reply index.html:6282]: Restored `kdenhartog-sybil1`.  Do what you will.

File: index.html [hunk 4]
```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
...
+Christopher Allen, Joe Andrieu, Antonio, Phil Archer, George Aristy, Baha, Juan
+Benet, BigBlueHat, Dan Bolser, Chris Boscolo, Pelle Braendgaard, Daniel Buchner,
+Daniel Burnett, By_caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
+Wayne Chang, Sam Curren, Hai Dang, Tim Daubenschütz, Oskar van Deventer, Kim
+Hamilton Duffy, Arnaud Durand, Ken Ebert, Veikko Eeva, Carson Farmer, Nikos
+Fotiou, Gabe, Gayan, Ryan Grant, Peter Grassberger, Adrian Gropper, Amy Guy,
+Daniel Hardman, Kyle Den Hartog, Philippe Le Hegaret, Ivan Herman, Michael
+Herman, Alen Horvat, Dave Huseby, Marcel Jackisch, Mike Jones, Andrew Jones, Tom
+Jones, jonnycrunch, Ted Thibodeau Jr, Gregg Kellogg, Michael Klein, Paul
+Knowles, David I. Lehn, Charles Lehner, Michael Lodder, Dave Longley, Tobias
```
Comment (clehner) [inline index.html:6272]:
```suggestion
Knowles, David I. Lehn, Charles E. Lehner, Michael Lodder, Dave Longley, Tobias
```

File: index.html [hunk 5]
```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
 Christopher Allen, Heather Vescent, and Wayne Chang.
     </p>
 
+    <p>
+The Working Group thanks the following individuals for their contributions to
+this specification (in alphabetical order): Denis Ah-Kang, Nacho Alamillo,
+Christopher Allen, Joe Andrieu, Antonio, Phil Archer, George Aristy, Baha, Juan
+Benet, BigBlueHat, Dan Bolser, Chris Boscolo, Pelle Braendgaard, Daniel Buchner,
+Daniel Burnett, By_caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
```
Comment (bumblefudge) [inline index.html:6265]: Oops I think there's a people.json somewhere I need to update.  Can I get by_caballero changed to Juan Caballero or do I have to open a separate PR?

Comment (bumblefudge) [inline index.html:6265]: By_caballero --> Juan Caballero plz

Comment (msporny) [inline index.html:6265]:
```suggestion
Daniel Burnett, Juan Caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
```

**Comment by msporny** [comment]
> I support folks who significantly contribute being added to the list of Editors and Authors, but need to be convinced that it is appropriate to remove someone who was named an Editor by the chairs.

The Chairs have decided that Drummond will remain as an Editor on the specification and have approved the rest of the Editors list. I have implemented that decision in the latest PR.

**Comment by dlongley** [comment]
> The Chairs have decided that Drummond will remain as an Editor on the specification and have approved the rest of the Editors list. I have implemented that decision in the latest PR.

Regarding the authors' list:

Based on core features contributed to the spec, I think the author list order should be:

Manu Sporny, Dave Longley, Markus Sabadello, Drummond Reed, Orie Steele, Christopher Allen

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: I added the `@` to all the Github names and sorted those into the alphabetical list (sorting them as if they were a last name), which is what I think you were asking for.

Implemented in ae5c53ca20e35ca81f2471515f423567cb89575c.

Not your perfect world where everyone has their Github handle by their name... it may make the list less readable AND will make the W3C link checker take *forever* AND I'm a bit loathe to point upwards of 50+ links to Github from a W3C spec AND I'd have to rewrite/debug some code instead of focusing on getting this Proposed Rec out... but getting 3 out of 4 editorial requests ain't bad. :)

**Comment by msporny** [comment]
@dlongley wrote:
> Based on core features contributed to the spec, I think the author list order should be...

@brentzundel emoji'd:
> :+1: 

I don't know if @dlongley is objecting, and I don't know if @brentzundel is expressing a Chair position... and considering it's the weekend and I don't know if the other Chair is available and given that I'm preparing the Proposed Recommendation right now.

I am going to merge over both of you. Please raise it on the call if that's an issue.

**Comment by dlongley** [comment]
Yeah, I think it would be good to make the author list order I mentioned before final Proposed Rec. I'll mention it on the call if it doesn't go in before then.

**Comment by msporny** [comment]
I have removed the `cr-comment-resolved-implicit` marker. @brentzundel and @burnburn, I noted that I won't be engaging on this issue any more; I said my piece. I expect that both Chairs will be processing the concern moving forward.

I will implement the final Chair decision, just let me know what that will be.
